src/components/RichTextEditor/BlockStyleButton.tsx

```diff
@@ -17,30 +19,26 @@ export const BlockStyleButton: FC<BlockStyleButtonProps> = ({ blockType, childre
         return null;
     }
 
-    const [{ context }, send] = useActor(machineRef);
-    const { editorState } = context;
+    const editor = useSlateStatic();
+    const [blockTypeIsActive] = Editor.nodes(editor, {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        match: (n) => Element.isElement(n) && n.type === blockType,
```

Comment:
What is `n`? Could you put a more meaningful name?

Reply:
738381e